lib/config.ts

```diff
@@ -319,7 +269,7 @@ export interface Config {
    * `beforeLaunch` is run, and before any driver is set up. If this is
    * specified, both capabilities and multiCapabilities will be ignored.
    */
-  getMultiCapabilities?: any;
+  getMultiCapabilities?: () => q.IWhenable<Array<Capabilities>>;
```

Comment:
It may or may not be a promise, and I'm not sure but I think `Promise` is a class not an interface, but I'll check and figure something better out